strongbox.go

```diff
@@ -65,16 +65,17 @@ func main() {
 
 	// Set up keyring file name
 	var home string
-	u, err := user.Current()
-	if err != nil {
-		// Possibly compiled without CGO and syscall isn't implemented,
-		// try to grab the environment variable
+	home = os.Getenv("STRONGBOX_HOME")
+	if home == "" {
 		home = os.Getenv("HOME")
-		if home == "" {
-			log.Fatal("Could not call os/user.Current() or find $HOME. Please recompile with CGO enabled or set $HOME")
+	}
+	if home == "" {
+		u, err := user.Current()
```

Comment:
You are changing the order of execution, I think its better to use stdlib before falling back to env vars.

Reply:
Yep, I wasn't too sure about this one. Since I needed to check for the custom env var, changing the order here made the code a bit clearer imo.

I'll revert the last commit